src/library-authoring/routes.ts

```diff
@@ -17,23 +17,21 @@ export const BASE_ROUTE = '/library/:libraryId';
 
 export const ROUTES = {
   // LibraryAuthoringPage routes:
-  // * Components tab, with an optionally selected componentId in the sidebar.
-  COMPONENTS: '/components/:componentId?',
+  // * Components tab, with an optionally selected component in the sidebar.
+  COMPONENTS: '/components/:selectedItemId?',
   // * Collections tab, with an optionally selected collectionId in the sidebar.
-  COLLECTIONS: '/collections/:collectionId?',
+  COLLECTIONS: '/collections/:selectedItemId?',
   // * Sections tab, with an optionally selected sectionId in the sidebar.
-  SECTIONS: '/sections/:sectionId?',
+  SECTIONS: '/sections/:selectedItemId?',
   // * Subsections tab, with an optionally selected subsectionId in the sidebar.
-  SUBSECTIONS: '/subsections/:subsectionId?',
+  SUBSECTIONS: '/subsections/:selectedItemId?',
   // * Units tab, with an optionally selected unitId in the sidebar.
-  UNITS: '/units/:unitId?',
-  // * All Content tab, with an optionally selected componentId in the sidebar.
-  COMPONENT: '/component/:componentId',
```

Comment:
Removed this route to use the default `HOME: ':selectedItemId?' since we don't have a dedicated page for components. We could use it in the future for the editor, though.